content/packages-and-modules/getting-packages-from-the-registry/searching-for-and-choosing-packages-to-download.mdx

```diff
@@ -4,39 +4,21 @@ redirect_from:
   - /getting-started/searching-for-packages
 ---
 
-You can use the npm search bar to find packages to use in your projects. npm search uses npms and the npms analyzer; for more information on both, see https://npms.io/about.
+You can use the npm search bar to find packages to use in your projects. The search is performed using content from the package's title, description, readme, and keywords and is powered by [opensearch](https://opensearch.org/). Search results are displayed based on keyword matching from the package's title, description, readme, and keywords. No subjective ranking criteria are applied, aiming to provide a neutral stance towards all (non-malicious, non-spam) packages.
```

Comment:
Well, we do apply a little subjective ranking to boost actively used packages above spammy or entirely new packages.